tests/datasets/api.yml

```diff
@@ -0,0 +1,160 @@
+---
+# Empty these tables
+cache:
+config:
+conversation:
+pconfig:
+photo:
+workerqueue:
+mail:
+
+# Populate tables with test data
+user:
+    -
+        uid: 42
+        username: Test user
+        nickname: selfcontact
+        verified: true
+        password: $2y$10$DLRNTRmJgKe1cSrFJ5Jb0edCqvXlA9sh/RHdSnfxjbR.04yZRm4Qm
+        theme: frio
+
+contact:
+    -
+        id: 42
+        uid: 42
+        name: Self contact
+        nick: selfcontact
+        self: true
+        nurl: http://localhost/profile/selfcontact
+        url: http://localhost/profile/selfcontact
+        about: User used in tests
+        pending: false
+        blocked: false
+        rel: 1
+        network: dfrn
+    -
+        id: 43
+        uid: 0
+        # Having the same name and nick allows us to test
+        # the fallback to api_get_nick() in api_get_user()
+        name: othercontact
+        nick: othercontact
+        self: false
+        nurl: http://localhost/profile/othercontact
+        url: http://localhost/profile/othercontact
+        pending: false
+        blocked: false
+        rel: 2
```

Comment:
I don't think you can have a public contact (`uid = 0`) with a two-way relationship. The problem is that we don't know who this contact would have a relationship with without a `uid` set.

Reply:
I'm not sure why I added this. I think it was in order to pass some condition in a tested function where I needed `rel` to be 2 or 3, but I can't find it anymore.
What would be the correct value here?

Reply:
`uid=42` or `rel=0`.